README.md

```diff
@@ -91,22 +95,25 @@ Run worker from local project using Docker
 
 ```
 USAGE
-  $ aio asset-compute run-worker FILE RENDITION
+  $ aio asset-compute run-worker [FILE] [RENDITION] [-v] [--version] [-a <value>] [-d <value> | -P <value> | -p
+    <value>]
 
 ARGUMENTS
   FILE       Path to input file for worker
-
   RENDITION  Path where to create output rendition.
              Single file for single rendition, or directory to create multiple renditions, in which case the full
              parameter json including rendition names must be provided using --data.
 
-OPTIONS
-  -P, --paramFile=paramFile  Path to parameter json file.
-  -a, --action=action        Worker to run. Use action name from manifest. Not required if there is only one.
-  -d, --data=data            Complete input parameters as JSON string. Allows multiple renditions.
-  -p, --param=param          <key> <value> - Set parameters for rendition, can be used multiple times
-  -v, --verbose              Verbose output
-  --version                  Show version
+FLAGS
+  -P, --paramFile=<value>  Path to parameter json file.
+  -a, --action=<value>     Worker to run. Use action name from manifest. Not required if there is only one.
+  -d, --data=<value>       Complete input parameters as JSON string. Allows multiple renditions.
+  -p, --param=<value>...   <key> <value> - Set parameters for rendition, can be used multiple times
```

Comment:
Do we really need the lower case too? `--param` is fine, but having both `-p` and `-P` is maybe a bit much?

Reply:
this is straight from here: https://github.com/adobe/aio-cli-plugin-asset-compute/blob/master/src/commands/asset-compute/run-worker.js#L213 ... nothing to do with this pr except maybe you are seeing some of the unused options for the first time.